packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -24,6 +39,177 @@ import {SparkPlugin, Page} from '@ciscospark/spark-core';
  * @name Memberships
  */
 const Memberships = SparkPlugin.extend({
+  /**
+   * Connect to the web socket to listen to incoming messages.
+   * @returns {Promise}
+   */
+  listen() {
+    // Create a common envelope that we will wrap all events in
+    this.eventEnvelope = {
+      resource: SDK_EVENT_RESOURCE.MEMBERSHIPS,
+      // id -- webhook id concept does not correlate to SDK socket event
+      // name -- webhook name concept does not correlate to SDK socket event
+      // targetUrl -- targetUrl concept does not correlate to SDK socket event
+      // secret -- secret concept does not correlate to SDK socket event
+      ownedBy: SDK_EVENT_OWNER.CREATOR,
+      status: SDK_EVENT_STATUS.ACTIVE,
+      created: new Date().toISOString(),
+      data: {}
+    };
+    const that = this;
+
+
+    that.spark.people.get('me')
+      .then((person) => {
+        that.eventEnvelope.createdBy = person.id;
+        that.eventEnvelope.orgId = person.orgId;
+      }).catch((e) => {
+        console.error(`Unable to get person info for memberships \
+          event envelope ${e.message}`);
+      });
+
+    // Register to listen to events
+    return this.spark.internal.mercury.connect()
+      .then(() => this.listenTo(this.spark.internal.mercury,
```

Comment:
The carriage returns here will probably get flagged by linting.

Reply:
Actually, I added it because the linter complained about the line length.   (This is actually what your code looks like in messages.js)

Hopefully the new multi-line brace function will make all parties happy.